### DISCUSSION TITLE
Student Account Application

### DISCUSSION THREAD

**Discussion by gifari1998** [community/community#48075] [GitHub Education]
### Select Topic Area

Question

### Body

While applying for joining the GitHub Global campus, it's frequently showing the same thing as following "Attendees of this school must verify and use their academic email address." I don't know how to fix it. I am using my academic email address every time. Can anyone help me fix this problem?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @gifari1998! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
